imports/plugins/included/payments-stripe/server/methods/stripe.js

```diff
@@ -28,17 +25,33 @@ function unformatFromStripe(amount) {
   return (amount / 100);
 }
 
+export const utils = {};
+
+utils.getStripeApi = function (paymentPackageId) {
+  const stripePackage = Packages.findOne(paymentPackageId);
+  const stripeKey = stripePackage.settings.api_key || stripePackage.settings.connectAuth.access_token;
+  return stripeKey;
+};
+
+/**
+ * @summary Capture the results of a previous charge
+ * @param {object} paymentMethod - Object containing info about the previous transaction
+ * @returns {object} Object indicating the result, saved = true means success
+ */
 function stripeCaptureCharge(paymentMethod) {
   let result;
   const captureDetails = {
     amount: formatForStripe(paymentMethod.amount)
   };
 
+
+  const stripeKey = utils.getStripeApi(paymentMethod.paymentPackageId);
+  const stripe = stripeNpm(stripeKey);
+
   try {
-    const captureResult = StripeApi.methods.captureCharge.call({
-      transactionId: paymentMethod.transactionId,
-      captureDetails: captureDetails
-    });
+    const capturePromise = stripe.charges.capture(paymentMethod.transactionId, captureDetails);
```

Comment:
Do you think we should change my current `createCharges` method to use Promises instead of async/await? Feels weird to me to do this in different ways within the same plugin.